codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/util/Smithy.kt

```diff
@@ -100,7 +100,11 @@ fun ServiceShape.hasEventStreamOperations(model: Model): Boolean =
 
 fun Shape.shouldRedact(model: Model): Boolean =
     when (this) {
-        is MemberShape -> model.expectShape(this.target).shouldRedact(model) || model.expectShape(this.container).shouldRedact(model)
+        is MemberShape ->
+            model.expectShape(this.target).shouldRedact(model) ||
```

Comment:
**style** You don't need `this.` qualifier, it should recognize `target` and `container` as belonging to `this`

Reply:
Removed in https://github.com/smithy-lang/smithy-rs/pull/3730/commits/6f9029d63344301677ebb7c2e15ea35f5944d667